src/index.html

```diff
@@ -17,11 +17,449 @@
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="menu__item">
+            <a
+              class="menu__link"
+              href="gadgets"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="menu__item">
+            <a
+              class="menu__link"
+              href="tablets"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="menu__item">
+            <a
+              class="menu__link"
+              href="photo"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="menu__item">
+            <a
+              class="menu__link"
+              href="video"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__photo"></div>
+
+        <a
+          href="#"
+          class="card__name"
+        >
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </a>
+
+        <span class="card__product">Product code: 195434</span>
+
+        <div class="card__reviews">
+          <div class="stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <a
+            href="#"
+            class="reviews"
+          >
+            Reviews: 5
+          </a>
+        </div>
+
+        <div class="price">
+          <span class="price__name">Price:</span>
+          <span class="price__number">$2,199</span>
+        </div>
+
+        <button
+          class="card__button"
+          data-qa="card-hover"
+        >
+          Buy
+        </button>
+      </div>
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__photo"></div>
+
+        <a
+          href="#"
+          class="card__name"
+        >
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </a>
+
+        <span class="card__product">Product code: 195434</span>
+
+        <div class="card__reviews">
+          <div class="stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <a
+            href="#"
+            class="reviews"
+          >
+            Reviews: 5
+          </a>
+        </div>
+
+        <div class="price">
+          <span class="price__name">Price:</span>
+          <span class="price__number">$2,199</span>
+        </div>
+
+        <button
+          class="card__button"
+          data-qa="hover"
+        >
```

Comment:
The 'data-qa' attribute should be 'card-hover', not 'hover'. This is important for testing purposes.